### PR TITLE
docs: notificaiton blog

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 476e234</samp>

Add a blog post in Chinese about the Notification stack feature. The post introduces and illustrates the feature, which allows users to customize the stacking behavior of Notification messages.

### <samp>🤖 Generated by Copilot at 476e234</samp>

* Add a new blog post in Chinese about the Notification stack feature ([link](https://github.com/ant-design/ant-design/pull/45199/files?diff=unified&w=0#diff-efea815a4fa9f01124e39ac002cfbdd32a10711befc621150e15304cb05a8e84R1-R43))

### CODE REVIEW

File: docs/blog/notification-stack.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+当然也不是只要将高度直接相等就结束了，别忘了我们的 Notification 是支持展开的，在展开时所有的消息框都应该恢复到他原本的大小，并且由原本的高度来计算展开时的偏移量。但是宽高都已经被修改了，我们需要如何还原呢？为了实现这个新特性，我们在原本的消息框外层新增了一层 div，作为消息框的容器，来承担尺寸变化的职责。而原本的消息框则只保留了内容部分，保证自身的高度不受容器的影响。而宽度则是容器由 `scaleX` 来变化，这就避免了宽度变化导致文字换行撑高容器高度的情况。在这层结构下，我们既能得到正确的内容尺寸，又可以随意修改容器的尺寸，而不影响内容的排版，一举两得。 ![image.png](https://intranetproxy.alipay.com/skylark/lark/0/2023/png/43956295/1696667649145-5f6c249f-139c-4ba4-9090-7bd7dfbc846c.png#clientId=u1b5ab9db-426e-4&from=paste&height=262&id=u4ff934fa&originHeight=262&originWidth=408&originalType=binary&ratio=1&rotation=0&showTitle=false&size=30058&status=done&style=none&taskId=ue20fd8d6-a2ca-46e0-baa2-d79f23ee9bd&title=&width=408)
+
+#### 定位方法
+
+在定位上我们选择了最简单的方法，将所有消息框都绝对定位，展开时则是计算每一个消息框的高度，将其定位偏移量累积起来，达到目前这样流畅的动画效果。如果不考虑动画效果的话还会有其他的布局方法。比如在实现过程中我们就采用过 flex 布局，将展开时的顺序排列先实现好，然后再通过负的 `margin` 来实现堆叠效果。这同样是一种思路，但实际上会有一些动画效果上的问题：在新的消息出现时，所有下方的消息都会有一个瞬间的位移来给新消息让位。这个问题和一些其他的定位问题会融合到一起最后比较难完美解决，所以我们最终还是采用了最简单的方式——绝对定位，所有的偏移量都由我们手动计算。
+
+### 总结
+
+这个新特性是 antd 团队为了提升组件的灵活度与时髦度的一次尝试，希望能够带给大家更好的体验～同样因为是新的特性，如果有遇到使用问题或者 Bug 的话欢迎到 antd 的 issue 区提出。最后希望大家喜欢这个灵动的 Notification～
```
Comment (kiner-tang) [inline docs/blog/notification-stack.zh-CN.md:45]:
```suggestion
这个新特性是 antd 团队为了提升组件的灵活度与时髦度的一次尝试，希望能够带给大家更好的体验~。同样因为是新的特性，如果有遇到使用问题或者 Bug 的话欢迎到 antd 的 issue 区提出。最后希望大家喜欢这个灵动的 Notification～。
```

**Review by yoyo837 — APPROVED** [review]
.

File: docs/blog/notification-stack.en-US.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+![image.png](https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*1cZGRKgxucsAAAAAAAAAAAAADrJ8AQ/original)
+
+#### Positioning
+
+In terms of positioning, we chose the simplest method, absolutely positioning all message boxes. When expanding, we calculate the height of each message box and accumulate its positioning offsets to achieve the current smooth animation effect.
+
+If animation effects are not considered, there are other layout methods. For example, during the implementation process, we used flex layout to implement the order of expansion first, and then used negative `margin` to achieve the stacking effect. This is also an idea, but in fact there will be some problems with the animation effect: when a new message appears, all the messages below will have an instant displacement to make way for the new message. This problem and some other positioning problems will be merged together and ultimately difficult to solve perfectly, so we finally adopted the simplest method - absolute positioning, and all offsets were calculated manually.
+
+### 总结
```
Comment (kiner-tang) [inline docs/blog/notification-stack.en-US.md:51]:
```suggestion
### Summary
```

**Review by kiner-tang — APPROVED** [review]

**Comment by kiner-tang** [comment]
@afc163 有一个中文漏改了：
- https://github.com/ant-design/ant-design/pull/45200

**Pull request merged**